experimental/iree_profiler/library.py

```diff
@@ -0,0 +1,294 @@
+import enum
+from enum import auto as enum_auto
+import re
+import numpy as np
+from collections import namedtuple
+
+###################################################################################################
+# This file contains library of enumerations and classes used to build operation descritpions.
+# The operation descriptions are used to generate MLIR source files, performance tuning configuration,
+# reference implementations, and numpy input/output files.
+###################################################################################################
+
+
+# Architecure types
+###################################################################################################
+class ArchType(enum.Enum):
+  Cpu = enum_auto()
```

Comment:
Nit, I would probably leave it as Cpu = enum.auto, then just import enum (not introduce the enum_auto alias)

Reply:
Applied this comment. I have experienced collisons and that is why imported as an alias, but if we don't run into issues with `enum.auto()`, we can avoid the alias.